### PR TITLE
Add first draft of stylesheet

### LINKED ISSUE #267: We should have a style guide for LWKD
It would help contributors to LWKD to have a style guide.

While we can borrow some things (mainly about project/tool naming) from the [Docs style guide](https://kubernetes.io/docs/contribute/style/style-guide/), we need some guide materials that are peculiar to LWKD.

### PR DESCRIPTION
**Pull request by jberkus**
Fixes #267 

@sreeram-venkitesh @fykaa

### CODE REVIEW

File: STYLESHEET.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+```
+
+Use code formatting for variable names, commands, and switches, and optionally use it for feature gates.
+
+```
+* `nominalConcurrencyShares` [can be zero](https://github.com/kubernetes/kubernetes/pull/123001)
+* `--node-labels` has been around for 28 releases, [maybe it's not alpha anymore](https://github.com/kubernetes/kubernetes/pull/123061)
+```
+
+Version numbers can be reported as either bare or with a "v", whatever seems appropriate.
```
Comment (mfahlandt) [inline STYLESHEET.md:55]:
```suggestion
Version numbers can be reported as either bare or with a "v", whatever seems appropriate. In doubt use the formate the project/subproject is using.
```

File: STYLESHEET.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+* [etcd to v3.5.10](https://github.com/kubernetes/kubernetes/pull/121573)
+* [go to 1.21.7](https://github.com/kubernetes/kubernetes/pull/123230) in 1.26 through 1.29, and [to 1.22 in 1.30](https://github.com/kubernetes/kubernetes/pull/123217)
+
+```
+
+### Developer News
+
+Developer News should share with contributors the kinds of announcements that are also posted to `kubernetes/dev` and `#kubernetes-org-members`.
+
+In general, you should not repeat any news that has already appeared in LWKD. The main exception to this is major, project-spanning changes which have been hard to get people to pay attention to (i.e. moving the container images), in which case you might want to run those 2-3 times.
```
Comment (mfahlandt) [inline STYLESHEET.md:137]:
```suggestion
In general, you should not repeat any news that has already appeared in LWKD. The main exception to this is major, project-spanning changes or event related topcis like contributor summit information, which have been hard to get people to pay attention to (i.e. moving the container images). For those cases you might want to run them 2-3 times.
```

**Review by mfahlandt — COMMENTED** [review]
Some suggestions

File: STYLESHEET.md (hunk 1 above)
Comment (jberkus) [reply STYLESHEET.md:55]: Can you untypo the suggestion?

**Comment by jberkus** [comment]
Merging this because the team urgently needs it

**Pull request merged**